src/main/java/com/fincons/taskmanager/controller/TaskController.java

```diff
@@ -12,45 +13,163 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @RestController
 @CrossOrigin("*")
 @RequestMapping("/task-manager")
 public class TaskController {
 
     @Autowired
     private TaskService taskService;
-
     @Autowired
-    private TaskMapper taskMapper;
+    public TaskMapper modelMapperTask;
```

Comment:
why is this public?

Reply:
Perché lo uso nei test, o per lo meno lo userò

Reply:
things should not be public in the main program, we are not going to publish the tests when we deploy